lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -662,14 +662,31 @@ extension DomElementExtension on DomElement {
   double? get tabIndex => _tabIndex?.toDartDouble;
 
   @JS('focus')
-  external JSVoid _focus(JSAny options);
+  external JSVoid _focusWithOptions(JSAny options);
 
-  void focus({bool? preventScroll, bool? focusVisible}) {
-    final Map<String, bool> options = <String, bool>{
-      if (preventScroll != null) 'preventScroll': preventScroll,
-      if (focusVisible != null) 'focusVisible': focusVisible,
-    };
-    _focus(options.toJSAnyDeep);
+  @JS('focus')
+  external JSVoid _focus();
+
+  /// Calls DOM `Element.focus`.
+  ///
+  /// If [preventScroll] is true, prevents the browser from scrolling to the
+  /// focused element.
+  ///
+  /// IMPORTANT: [preventScroll] defaults to `true`, which is the exact opposite
+  /// of the browser default. This is because by default, the Flutter framework
```

Comment:
> [preventScroll] defaults to `true`, which is the exact opposite of the browser default.

Please, don't do this, the JS-interop layer should be as similar as the MDN as possible. If we ever replace `dom.dart` by `package:web`, all this would immediately break :/

Reply:
Technically, this is not an interop method for `focus`. The methods that bind to `Element.focus` are `_focus` and `_focusWithOptions`. However, the name does clash with the standard web name. I can rename it to something else, like `focusSafely`. Would that work?

Reply:
@yjbanov Yes. I'm fine with `focusSafely` or a `focus` that behaves the same as the one in MDN. Anything that makes it not compatible with the standard method is fine :P

PS: this *is* a JS-interop method, even if it's just a trampoline to call the `external` ones.

(My argument stands, if we were to replace this with `package:web`, the code would be doing the opposite to what we want, because [this](https://pub.dev/documentation/web/0.5.1/web/HTMLElement/focus.html) would compile, but behave erroneously.)

Reply:
`focusWithoutScroll` or something :P

Reply:
Done.